packages/ava/src/utils/types.ts

```diff
@@ -0,0 +1,18 @@
+import type { AntVSpec } from '@antv/antv-spec';
+
+/**
+ * One row(record) of data in JSON.
+ */
+export type DataRow = Record<string, any>;
```

Comment:
G2/NTV/LI call it Datum
![image](https://user-images.githubusercontent.com/15177000/201523123-305403ee-5679-4858-8f6b-953b9181a139.png)

![image](https://user-images.githubusercontent.com/15177000/201523137-51d7c8dd-7835-407f-94b0-05c6087149ab.png)

Reply:
DataRow is more clear and Datum maybe more accurate, no best choice...but it could be datum for consistency.

but the value could be other than 'string | number'? like boolean? date? 
'string | number' seems it is 'CsvDatum' in default.

Reply:
cc @BBSQQ

Reply:
It'meaning datum not csv datum in ntv, will extend bool types later.

Reply:
fixed